### PR TITLE
Random upgrades

### PR DESCRIPTION
**Pull request by NeilFraser**
Procrastination is a wonderful way to do cleaning...

### CODE REVIEW

File: appengine/storage.js [hunk 1]
```diff
@@ -169,10 +169,10 @@ BlocklyStorage.monitorChanges_ = function(workspace) {
     var xmlText = Blockly.Xml.domToText(xmlDom);
     if (startXmlText != xmlText) {
       window.location.hash = '';
-      workspace.removeChangeListener(bindData);
+      workspace.removeChangeListener(change);
     }
   }
-  var bindData = workspace.addChangeListener(change);
+  workspace.addChangeListener(change);
```
Comment (AnmAtAnm) [inline appengine/storage.js]: While true in practice, the `addChangeListener(..)` documentation does not guarantee the returned function is the same as the passed function.

File: core/block_svg.js [hunk 2]
```diff
@@ -986,10 +986,9 @@ Blockly.BlockSvg.prototype.setBorderColour_ = function(rgb, colourTertiary) {
  * Sets the colour of shadow blocks.
  * @param {!string} rgb Primary colour of the block.
  * @param {?string} colourSecondary Colour for shadow block.
- * @return {!string} hexColour The background color of the block.
+ * @return {!string} hexColour The background colour of the block.
```
Comment (AnmAtAnm) [inline core/block_svg.js]: Remove `hexColour`. Return values don't have names.

Comment (NeilFraser) [reply core/block_svg.js]: Fixed this and 65 other @return annotations.

File: core/gesture.js [hunk 3]
```diff
@@ -954,3 +954,18 @@ Blockly.Gesture.prototype.getInsertionMarkers = function() {
   }
   return [];
 };
+
+/**
+ * Is a drag or other gesture currently in progress on any workspace?
+ * @return {boolean} True if gesture is occurring.
+ */
+Blockly.Gesture.inProgress = function() {
+  // If a drag is occurring, then it involves a selected block on a workspace.
```
Comment (rachel-fenichel) [inline core/gesture.js]: What about a workspace drag?

**Comment by rachel-fenichel** [comment]
FYI we can enforce return vs returns with the `prefer` option in the `valid-jsdoc` rule: https://eslint.org/docs/rules/valid-jsdoc#options

If it's important we can probably enforce capitals at the start of param/return descriptions with `matchDescription`, documented in the same place.

File: core/gesture.js (hunk 3 above)
Comment (NeilFraser) [reply core/gesture.js]: Fixed, now searching all workspaces.

File: appengine/storage.js (hunk 1 above)
Comment (NeilFraser) [reply appengine/storage.js]: Nowhere in Blockly are we still using the returned function (this was the last straggler).  I've changed the docs to indicate that the return value is obsolete and to be ignored.

**Pull request merged**